client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -408,13 +412,14 @@ private void checkAckLevels(long ackLevel, long previousAckLevel) {
 
         @Override
         public void processingFailure(Exception error) {
+            log.info("Received processingFailure with {}", error);
             failConnection(error);
         }
 
         @Override
         public void authTokenCheckFailed(WireCommands.AuthTokenCheckFailed authTokenCheckFailed) {
-            log.warn("Auth failed {}", authTokenCheckFailed);
-            failConnection(new AuthenticationException(authTokenCheckFailed.toString()));
+            log.info("Received authTokenCheckFailed with {}", authTokenCheckFailed);
```

Comment:
Why is this info and not warn? If it is an error, but it is a recoverable error, then it should be warn level.

Reply:
When looking to make the suggested change, I realized that the `failConnection()` method was already logging the exception. So, I removed the log entries from the methods that invoke `failConnection` in order to avoid redundant logging. The `failConnection()` was logging at an info level, which I have now modified to `warn`. Hope that helps.